src/hsmm_mvpy/visu.py

```diff
@@ -174,25 +172,19 @@ def plot_topo_timecourse(channels, estimated, channel_position, init, time_step=
     if not return_ax:
         ax.spines['top'].set_visible(False)
         ax.spines['right'].set_visible(False)
-        ax.set_ylim(0-yoffset, n_iter-1+yoffset)
+        ax.set_ylim(0, n_iter) #-1
         ax.set_xlabel(xlabel)
         if title:
             ax.set_title(title)
         if np.any(max_time) == None and np.any(times_to_display) == None:
             ax.set_xlim(0, np.nanmax(times)+np.nanmax(times)/10)
     if return_ax:
-        ax.set_ylim(0-yoffset, n_iter-1+yoffset)
+        ax.set_ylim(0, n_iter) #-1
         return ax
     else:
         plt.show()    
 
-def plot_components_sensor(hmp_data, positions):
-    from mne.viz import plot_topomap
-    fig, ax = plt.subplots(1,len(hmp_data.attrs['components'].component))
-    for comp in hmp_data.attrs['components'].component:
-        plot_topomap(hmp_data.attrs['components'].values[:,comp], positions, axes=ax[comp], show=False, cmap='Spectral_r')
-    plt.show()
```

Comment:
Why did you remove this? Is this a version conflict?

Reply:
this shouldn't have been removed, I'll do a new commit.